src/sdk/Transcription/Conversation.ts

```diff
@@ -895,7 +863,7 @@ export class ConversationImpl extends Conversation implements IDisposable {
         }
     }
 
-    private onTranslationReceived = (r: ConversationRecognizer, e: ConversationReceivedTranslationEventArgs): void => {
+    public onTranslationReceived = (r: ConversationRecognizer, e: ConversationReceivedTranslationEventArgs): void => {
```

Comment:
>onTranslationReceived [](http://example.com/codeflow?start=11&length=21)

Why are these now public on the SDK surface?

Reply:
They're part of ConversationImpl, so they're only available after recasting, much like the recognizerImpl methods.